lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -26,3 +26,8 @@ export type AccountInfo = {
     idTokenClaims?: TokenClaims & { [key: string]: string | number | string[] | object | undefined | unknown };
     nativeAccountId?: string;
 };
+
+export type ActiveAccountIds = {
+    homeAccountId: string;
+    localAccountId: string;
+};
```

Comment:
FYI could also be:

```suggestion
export type ActiveAccountIds = Pick<AccountInfo, "homeAccountId" | "localAccountId">;
```

Reply:
And I would suggest being a little more explicit, maybe `ActiveAccountIdKeys` or `ActiveAccountIdProperties`?